pkgs/applications/networking/remote/teamviewer/default.nix

```diff
@@ -6,6 +6,7 @@
 
 mkDerivation rec {
   pname = "teamviewer";
+  outputs = [ "out" "dev" ];
```

Comment:
```suggestion
```
dev is empty.

Reply:
It should contain the dev qtquickcontrols dependency

Reply:
I don't see it.

```ShellSession
$ nix build .#teamviewer.dev
....
$ tree result-dev/
 result-dev
└──  nix-support
    └──  propagated-build-inputs
```

Reply:
`propagated-build-inputs`

Reply:
Intersting 🤔